### PR TITLE
Feature Cart page

### PR DESCRIPTION
**Pull request by Avin008**
### Cart page Includes:

carts page

### CODE REVIEW

**Comment by CallMeSahu** [comment]
<title>Document</title> Change your title to "Cart". Rest everything is good.

File: pages/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.checkout-card .CTA__btn {
+  padding: 0.5rem 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0.6rem;
+  text-decoration: none;
+  font-weight: 600;
```
Comment (kotalwar-krishna) [inline pages/cart/cart.css:49]: for font weight use var if you are using repeatatively

**Review by kotalwar-krishna — APPROVED** [review]
all looks fine

**Review by SuyashTiwariCoder — APPROVED** [review]
Looks good to me , You can merge .

**Pull request merged**